apps/src/gamelab/GameLabSprite.js

```diff
@@ -251,10 +248,12 @@ var AABBops = function (p5Inst, type, target, callback) {
               callback.call(this, this, other);
           }
         }
-      else if(type === 'collide' || type === 'bounce')
+      else if(type === 'collide' || type === 'bounce' || type == 'bounceOff')
```

Comment:
Totally agree. Sadly, we already have a few other more significant changes in this function (since it is now stateful async), so I think that is a challenge we will have to live with.

The best way to mitigate the ongoing pain from this particulate change is to propose making this "bounceOff" change an official part of p5.play.